cfme/tests/control/test_alerts.py

```diff
@@ -185,6 +186,24 @@ def vm_crud(provider, vm_name, full_template):
     return VM.factory(vm_name, provider, template_name=full_template["name"])
 
 
+@pytest.fixture(scope="module")
+def ssh_appliance():
+    return SSHClient(
+        username=credentials['ssh']['username'],
+        password=credentials['ssh']['password'],
+        hostname=urlparse(store.base_url).netloc
```

Comment:
We shouldn't reach out to base_url as a different appliance could be on the stack.
Is there a reason why ```store.current_appliance.ssh_client``` cant be used here and negate the need for a fixture completely